spec.html

```diff
@@ -27046,58 +27055,48 @@ <h1>Hours, Minutes, Second, and Milliseconds</h1>
+        <h1>MakeTime ( _hour_, _min_, _sec_, _millisec_ )</h1>
+        <p>The abstract operation MakeTime calculates a number of milliseconds from its four arguments, which must be ECMAScript Number values. It performs the following steps:</p>
         <emu-alg>
-          1. If _hour_ is not finite or _min_ is not finite or _sec_ is not finite or _ms_ is not finite, return *NaN*.
+          1. If _hour_ is not finite or _min_ is not finite or _sec_ is not finite or _millisec_ is not finite, return *NaN*.
           1. Let _h_ be ! ToInteger(_hour_).
           1. Let _m_ be ! ToInteger(_min_).
           1. Let _s_ be ! ToInteger(_sec_).
-          1. Let _milli_ be ! ToInteger(_ms_).
-          1. Let _t_ be _h_ `*` msPerHour `+` _m_ `*` msPerMinute `+` _s_ `*` msPerSecond `+` _milli_, performing the arithmetic according to IEEE 754-2008 rules (that is, as if using the ECMAScript operators `*` and `+`).
+          1. Let _ms_ be ! ToInteger(_millisec_).
+          1. Let _t_ be _h_ `*` msPerHour `+` _m_ `*` msPerMinute `+` _s_ `*` msPerSecond `+` _ms_, performing the arithmetic according to IEEE 754-2008 rules (that is, as if using the ECMAScript operators `*` and `+`).
           1. Return _t_.
         </emu-alg>
       </emu-clause>
 
       <emu-clause id="sec-makeday" aoid="MakeDay">
         <h1>MakeDay ( _year_, _month_, _date_ )</h1>
-        <p>The abstract operation MakeDay calculates a number of days from its three arguments, which must be ECMAScript Number values. This operator functions as follows:</p>
+        <p>The abstract operation MakeDay calculates a number of days from its three arguments, which must be ECMAScript Number values. It performs the following steps:</p>
         <emu-alg>
           1. If _year_ is not finite or _month_ is not finite or _date_ is not finite, return *NaN*.
           1. Let _y_ be ! ToInteger(_year_).
           1. Let _m_ be ! ToInteger(_month_).
-          1. Let _dt_ be ! ToInteger(_date_).
-          1. Let _ym_ be _y_ + floor(_m_ / 12).
-          1. Let _mn_ be _m_ modulo 12.
-          1. Find a value _t_ such that YearFromTime(_t_) is _ym_ and MonthFromTime(_t_) is _mn_ and DateFromTime(_t_) is 1; but if this is not possible (because some argument is out of range), return *NaN*.
-          1. Return Day(_t_) + _dt_ - 1.
+          1. Let _d_ be ! ToInteger(_date_).
+          1. Let _wholeYears_ be _y_ `+` floor(_m_ / 12), performing the arithmetic according to IEEE 754-2008 rules (that is, as if using the ECMAScript `+` operator).
+          1. If _wholeYears_ is not finite, return *NaN*.
+          1. Let _monthWithinYear_ be _m_ modulo 12.
+          1. Let _inLeapYear_ be InLeapYear(TimeFromYear(_wholeYears_)).
+          1. Let _day_ be DayFromYear(_wholeYears_) `+` DaysWithinYearBeforeMonth(_monthWithinYear_, _inLeapYear_) `+` _d_ `-` 1, performing the arithmetic according to IEEE 754-2008 rules (that is, as if using the ECMAScript operators `+` and `-`).
```

Comment:
if you use the subscript `𝔽`, i don't think you need the "performing …" prose, since that makes them Numbers?

Reply:
You don't even need the subscript, since it's optional when the operands are Number values, which they presumably are, according to [5.2.5](https://tc39.es/ecma262/#sec-mathematical-operations). And the operators don't need backticks either.

Reply:
even better

Reply:
Cool; I'll update that. Should multiplication of Number values use `*` or `×`?